mods/cnc/maps/nod02b/map.yaml

```diff
@@ -0,0 +1,396 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: Invasion of Egypt (b)
+
+Description: GDI has kept a stranglehold on Egypt for many years.  Set up a forward attack base in your area.  To do this you must select your Mobile Construction Vehicle (MCV) and right click on it. From here you can begin to build a base. This area contains plenty of Tiberium, so establishing the base should be easy.
```

Comment:
You're missing a space (there should be two) between ```on it.``` and ```From``` in comparison to nod02a.